client/src/containers/LocationForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect, Link } from 'react-router-dom';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+import * as actions from '../actions';
+import { locationdata } from '../location_data';
```

Comment:
@hoslack You don't have to put this ({locationdata} in an object since you only have one function

Reply:
@MarlenAw That's the best way to import the data  Mario told me so.

Reply:
👍

Reply:
@hoslack what no you never even asked me about this mate :sweat_smile: . the reason you are doing this is because you are doing `exports.something` and not `module.exports`